big_tests/tests/service_mongoose_system_stats_SUITE.erl

```diff
@@ -59,34 +60,42 @@ init_per_group(_GroupName, Config) ->
 end_per_group(_GroupName, Config) ->
     Config.
 
-init_per_testcase(user_stats_are_reported_to_google_analytics_when_mim_starts, Config) ->
+init_per_testcase(system_stats_are_reported_to_google_analytics_when_mim_starts, Config) ->
     create_events_collection(),
-    enable_user_stats(mim()),
+    enable_system_stats(mim()),
     delete_prev_client_id(mim()),
     Config;
+init_per_testcase(system_stats_are_not_reported_when_not_allowed, Config) ->
+    create_events_collection(),
+    disable_system_stats(mim3()),
+    delete_prev_client_id(mim3()),
+    Config;
 init_per_testcase(_TestName, Config) ->
     create_events_collection(),
```

Comment:
How comes we create the events collection on `init_per_testcase`, but never call `clear_events_collection/0` anywhere? Seems like suspiciously asymmetric 😕

Reply:
added clear_events_collection at the end of all end_per_testcase